puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -15,5 +15,5 @@ def set_job_build_description() {
        build_description += "<a href=\"${link}\">${files_list[i]}</a><br/>"
     }
 
-    currentBuild.description = build_description
+    currentBuild.description += build_description
```

Comment:
I wonder if description is ever null

Reply:
You wonder correctly, and I had been testing an update to handle that.